.github/workflows/deploy-api-client.yml

```diff
@@ -0,0 +1,32 @@
+name: Deploy api and client
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - 'client/**'
+      - 'api/**'
+      - '.github/workflows/deploy-api-client.yml'
+      - 'docker-compose.yaml'
+  workflow_dispatch:
+
+env:
+  VITE_SESSION_SECRET: ${{ secrets.VITE_SESSION_SECRET }}
```

Comment:
Does the workflow for docker ssh use this? If so it's good practice to add the envvar as close to use as possible.